staff/francisco-fernandez/app-template/home.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Home Page</title>
+</head>
+<body>
+    <section>
+    <h1>My Website</h1>
```

Comment:
Include your links inside a ```<nav>```tag for better semantics and maybe inside an ```<ul>```or ```<ol>```

Reply:
and the anchors inside ```<li>``` too